snakeLadder.rb

```diff
@@ -0,0 +1,187 @@
+class SnakeLadder
+  attr_reader :board , :snakes , :ladders
+
+  def initialize
+    @board=[]
+    @snakes=[]
+    @ladders=[]
+    create_board
+    snake_placement
+    ladder_placement
+  end
+
+  def create_board
+    101.times do |i|
+      @board<<i
+    end
+  end
+  
+  def snake_placement
+
+    7.times  { @snakes<<rand(11..97) }
+    @snakes.sort!.uniq!
+    @snakes.each do |pos|
+      upper_limit=((pos/10))*10
+      pos % 10 == 0 ? upper_limit-=10 : upper_limit
+      @board[pos]=rand(3..upper_limit)
+    end
+
+  end
+
+
+  def ladder_placement
+
+    7.times { @ladders<<rand(3..90) }
+    @ladders.sort!.uniq!
+    @ladders.each do |pos|
+      lower_limit=((pos/10)+1)*10+1
+      pos % 10 == 0 ? lower_limit-=10 : lower_limit
+      @board[pos]=rand(lower_limit..97)
+    end
+
+  end
+
+  def is_ladder_used(obj)
```

Comment:
is this method return boolean value? if yes this this should be defined as `def is_ladder_used?(obj)`. Boolean methods should always be defied as question mark

Reply:
No , this method is not returning any bool value it is checking whether the current position has any ladder . if it has then it will change the current position of the player .

should I remove "is" from the method name ??

Reply:
yes remove `is`